index.js

```diff
@@ -49,3 +50,8 @@ module.exports = async function (fastify, opts) {
     reply.from(dest || '/', replyOpts)
   }
 }
+
+module.exports = fp(proxy, {
+  fastify: '^1.8.0',
+  name: 'fastify-http-proxy'
+})
```

Comment:
This is not correct. This plugin _should_ be encapsulated, and fastify-plugin removes encapsulation.

Reply:
Reverted, sorry about that